lib/xcpretty/formatters/simple.rb

```diff
@@ -89,6 +89,10 @@ def format_measuring_test(suite, test_case, time)
       )
     end
 
+    def format_phase_success(phase_name)
+      format(phase_name.capitalize, "Succeeded")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.